twilight-model/src/channel/mod.rs

```diff
@@ -81,9 +85,13 @@ pub struct Channel {
     /// This can be used to determine what fields *might* be available.
     #[serde(rename = "type")]
     pub kind: ChannelType,
-    /// ID of the last message sent in the channel.
+    /// For text channels, this is the ID of the last message sent in the
+    /// channel.
+    ///
+    /// For forum channels ([GuildForum](ChannelType::GuildForum)), this is
+    /// the ID of the last created thread in the forum.
```

Comment:
we probably don't need to link this:
```suggestion
    /// For forum channels, this is the ID of the last created thread in the
    /// forum.
```